packages/bolt-connection/src/rediscovery/rediscovery.js

```diff
@@ -18,7 +18,7 @@
  */
 import RoutingTable from './routing-table'
 import { RawRoutingTable } from '../bolt'
-import { newError, error, Session, ServerAddress } from 'neo4j-driver-core'
+import { newError, error, Session } from 'neo4j-driver-core'
```

Comment:
This was giving:
`SyntaxError: The requested module '../../core/index.ts' does not provide an export named 'ServerAddress'`

It seems that it's only used in a JSDoc comment here, and many other files reference it in a JSDoc without importing it so I assume this is fine.

Reply:
It's fine, the api documentation of the internals is not exported.